functions.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+export default_variant="jessie"
```

Comment:
why jessie and not stretch?

Reply:
Because were using jesse as the default now. However after this is merged, changing it will be trivial